docs/roadmap.md

```diff
@@ -2,10 +2,15 @@
 
 These are our upcoming features:
 
-- Support extracting runtime configurations from additional Cloud providers such as Azure & GCP
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Dynamic gathering and scan of Kubernetes cluster data through Kubernetes API
```

Comment:
```suggestion
- Kunerneter (dynamic gathering and scan of Kubernetes cluster data through Kubernetes API)
```
Shouldn't we keep 'Kuberneter'?
WDYT @nunoocx ?

Reply:
I agree with that approach, having Kuberneter in the description

Reply:
Yes I agree. My change proposal was just for the text within the parenthesis

Reply:
Done